docs/attributes.md

```diff
@@ -0,0 +1,94 @@
+### `#[StateId]`
+
+Link your event to its state(s) with the `StateId` attribute
+
+```php
+class YourEvent extends Event
+{
+    #[StateId(GameState::class)]
+    public int $game_id;
+
+    #[StateID(PlayerState::class)]
```

Comment:
@jdiddydave @inxilpro typo here with an all caps `ID`...unless it's not case sensitive? 🤔

Reply:
Ah thank you!